<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentsWidget</class>
 <widget class="QWidget" name="CommentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="atBeginButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="jle_resources.qrc">
           <normaloff>:/icons/Resources/pic/Button_ShiftUpUp.png</normaloff>:/icons/Resources/pic/Button_ShiftUpUp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="oneBackButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="jle_resources.qrc">
           <normaloff>:/icons/Resources/pic/Button_ShiftUp.png</normaloff>:/icons/Resources/pic/Button_ShiftUp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="oneForewardButton">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="jle_resources.qrc">
           <normaloff>:/icons/Resources/pic/Button_ShiftDown.png</normaloff>:/icons/Resources/pic/Button_ShiftDown.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="atEndButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="jle_resources.qrc">
           <normaloff>:/icons/Resources/pic/Button_ShiftDownDown.png</normaloff>:/icons/Resources/pic/Button_ShiftDownDown.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="jobListTableView">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
     <item>
      <widget class="QFrame" name="frInputElements">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>200</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="gbSiteConfig">
          <property name="title">
           <string>Site Information</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="3" column="0">
              <widget class="QLabel" name="lbOperator">
               <property name="text">
                <string>Operator</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="leArea"/>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="lbSiteId">
               <property name="text">
                <string>Site Number</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QSpinBox" name="sspinSiteNum"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="lbRunNumber">
               <property name="text">
                <string>Run Number</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="lbContractor">
               <property name="text">
                <string>Contractor</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lbClient">
               <property name="text">
                <string>Client</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="leClient"/>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="leContractor"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="leOperator"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lbArea">
               <property name="text">
                <string>Area</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lbSurveyId">
               <property name="text">
                <string>Survey ID</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="leSurveyId"/>
             </item>
             <item row="4" column="3">
              <widget class="QLineEdit" name="leSiteName">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="sspinLineNum"/>
             </item>
             <item row="4" column="2">
              <widget class="QLabel" name="lbSiteName">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Site Name</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QSpinBox" name="sspinRunNum"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="lbLineNumber">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Line Number</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="lbWeather">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Weather</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="leWeather">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="lbSiteNameRR">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Site Name Remote Reference</string>
               </property>
              </widget>
             </item>
             <item row="5" column="3">
              <widget class="QLineEdit" name="leSiteNameRR">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLabel" name="lbSiteNameRR_2">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Site Name EMAP</string>
               </property>
              </widget>
             </item>
             <item row="6" column="3">
              <widget class="QLineEdit" name="leSiteNameEMAP">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lbComment">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Comment</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leComment">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pbCommentChangeAll">
             <property name="text">
              <string>  Change Comments In All Jobs                                                          </string>
             </property>
             <property name="icon">
              <iconset resource="jle_resources.qrc">
               <normaloff>:/icons/Resources/pic/Button_OK.png</normaloff>:/icons/Resources/pic/Button_OK.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbProcMT">
          <property name="title">
           <string>Processing Instructions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lbProcessings">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Processings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leProcessings">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="pbConfigProcMT">
               <property name="text">
                <string>Automatically Configure MT Processing Instructions</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbConfigProcCSAMT">
               <property name="text">
                <string>Automatically Configure CSAMT Processing Instructions</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="jle_resources.qrc"/>
 </resources>
 <connections/>
</ui>
